docs/source/models/gbd2017/risk_attributable_causes/neonatal_preterm/index.rst

```diff
@@ -0,0 +1,23 @@
+.. _2017_cause_neonatal_preterm:
+
+====================================
+Neonatal preterm birth complications
+====================================
+
+Disease Description
+-------------------
+
+Case definition in GBD 2017
++++++++++++++++++++++++++++
+
+Preterm birth is defined as live birth before 37 completed weeks of gestation.
+In our analysis, we further break down this cause into three sub-categories of
```

Comment:
Maybe make clear that the 'our' is GBD, as opposed to us?

Reply:
Good point - I literally just copied and pasted this from the YLD Appendix. I'll make some minor edits to clarify.